optional/fetchmail/fetchmail.py

```diff
@@ -13,6 +13,7 @@
 
 FETCHMAIL = """
 fetchmail -N \
+    --idfile /data/.fetchids --uidl \
```

Comment:
I wouldn't use a hidden file here... why prefixing it with a dot? Some users may remove the folder thinking it's empty

Reply:
That is the suggested filename from the fetchmail documentation. I just followed their default file name. It indeed makes more sense to make the file not hidden (removing the leading dot).